### PR TITLE
Moved mechanical drawings

### PR DESCRIPTION
**Pull request by aallan**
Moved all the mechanical drawings to datasheets.raspberrypi.org.

### CODE REVIEW

File: hardware/raspberrypi/mechanical/README.md [hunk 1]
```diff
@@ -2,51 +2,47 @@
...
 ## Raspberry Pi Zero, Pi Zero W, Pi Zero WH (v1.3)
 
-- [Board](rpi_MECH_Zero_1p3.pdf)
-- [Case - Blank lid](rpi_MECH_Zero_case_blank.pdf)
-- [Case - GPIO lid](rpi_MECH_Zero_case_gpio.pdf)
-- [Case - Camera lid](rpi_MECH_Zero_case_camera.pdf)
-
-## Raspberry Pi Zero (v1.2)
-
-- [Board](rpi_MECH_Zero_1p2.pdf)
```
Comment (lurch) [inline hardware/raspberrypi/mechanical/README.md:41]: The reason that there were separate mechanical drawings for the Zero 1.2 and the Zero 1.3 is that the Zero1.2 doesn't have a camera-connector, but the Zero1.3 does. (although maybe this is a moot point, given that the Zero1.2 is no longer available for purchase?)

Comment (aallan) [reply hardware/raspberrypi/mechanical/README.md:41]: Ran this one past James A.

File: hardware/raspberrypi/mechanical/README.md [hunk 2]
```diff
@@ -2,51 +2,47 @@
...
+- [Board - Zero W](https://datasheets.raspberrypi.org/rpi/raspberry-pi-zero-w-mechanical-drawing.pdf)
+- [Board - Zero](https://datasheets.raspberrypi.org/rpi/raspberry-pi-zero-mechanical-drawing.pdf)
+- [Case - Blank lid](https://datasheets.raspberrypi.org/rpi/raspberry-pi-zero-case-mechanical-drawing.pdf)
+- [Case - GPIO lid](https://datasheets.raspberrypi.org/rpi/raspberry-pi-zero-case-with-gpio-mechanical-drawing.pdf)
+- [Case - Camera lid](https://datasheets.raspberrypi.org/rpi/raspberry-pi-zero-case-with-camera-mechanical-drawing.pdf)
```
Comment (lurch) [inline hardware/raspberrypi/mechanical/README.md:37]: Should these be in `/case/` rather than `/rpi/` ?

Comment (aallan) [reply hardware/raspberrypi/mechanical/README.md:37]: Yes they should, and they *are* on datasheets.raspberrypi.org. 🤦

Comment (aallan) [reply hardware/raspberrypi/mechanical/README.md:37]: Fixed, ee338cb.

**Pull request merged**